### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by andriy-shymkiv**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,19 @@
...
-  // write code here
+  const rentPrice = 40;
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
+
+  if (days >= 7) {
+    return days * rentPrice - sevenDaysDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
```
Comment (referee1988) [inline src/calculateRentalCost.js:39]:
```suggestion
  if (days >= 3) {
```
You have already done the check for 7 days above

**Review by referee1988 — APPROVED** [review]

File: src/calculateRentalCost.js (hunk 1 above)
Comment (referee1988) [reply src/calculateRentalCost.js:39]: Great job. There is a small recommendation that does not affect the correctness of the program.

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const sevenDaysDiscount = 50;
```
Comment (mykola-koval) [inline src/calculateRentalCost.js:32]: do not attach var names to days, use abstract names e.g. `smallDiscount` `largeDiscount`

File: src/calculateRentalCost.js (hunk 1 above)
Comment (mykola-koval) [reply src/calculateRentalCost.js:39]: agree

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**